kubernetes/samples/scripts/terraform/oke.create.sh

```diff
@@ -68,6 +69,60 @@ function createCluster () {
     terraform apply -auto-approve -var-file=${terraformVarDir}/${clusterTFVarsFile}.tfvars
 }
 
+function createRoleBindings () {
+kubectl -n kube-system create serviceaccount $okeclustername-sa
```

Comment:
add some indentation for the function body.

Reply:
fixed